### PR TITLE
Spectra v2.6.8

### PR DESCRIPTION
**Pull request by adi3890**
* Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
* Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.
* Fix: Form - Resolved an issue where the submit button is visible on the frontend when no preset from the variation picker is selected.
* Fix: Popup Builder - Resolved a conflict with Elementor that prevented users from opening a page in the Elementor page builder.

### CODE REVIEW

File: src/blocks/buttons-child/styling.js [hunk 1]
```diff
@@ -324,11 +327,19 @@ function styling( attributes, clientId, name ) {
...
+		] = {
+			'fill': iconHColor,
+		};
+	} else {
+		selectors[
+			'.uagb-buttons__outer-wrap .wp-block-button__link.uagb-buttons-repeater:hover .uagb-button__icon > svg'
+		] = {
+			'fill': hColor,
+		};
+	}
```
Comment (bsf-zanev) [inline src/blocks/buttons-child/styling.js:342]: These could be better approaches...

### Approach 1
```suggestion
	selectors[
		'.uagb-buttons__outer-wrap .wp-block-button__link.uagb-buttons-repeater:hover .uagb-button__icon > svg'
	] = {
		'fill': iconHColor || hColor,
	};
```

### Approach 2
```suggestion
	selectors[
		'.uagb-buttons__outer-wrap .wp-block-button__link.uagb-buttons-repeater:hover .uagb-button__icon > svg'
	] = {
		'fill': iconHColor ? iconHColor : hColor,
	};
```

File: src/blocks/container/edit.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ const UAGBContainer = ( props ) => {
 		clientId,
 		setAttributes,
 		name,
-		deviceType
+		deviceType,
+		context
```
Comment (bsf-zanev) [inline src/blocks/container/edit.js:44]: Always add trailing comma.

```suggestion
		context,
```

File: src/blocks/advanced-heading/edit.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ const UAGBAdvancedHeading = ( props ) => {
 		clientId,
 		name,
 		deviceType,
+		context
```
Comment (bsf-zanev) [inline src/blocks/advanced-heading/edit.js:27]: Always add trailing comma.

```suggestion
		context,
```

File: src/blocks/buttons-child/edit.js [hunk 4]
```diff
@@ -23,6 +23,7 @@ const ButtonsChildComponent = ( props ) => {
 		setAttributes,
 		name,
 		deviceType,
+		context
```
Comment (bsf-zanev) [inline src/blocks/buttons-child/edit.js:26]: Always add trailing comma.

```suggestion
		context,
```

File: src/blocks/image/edit.js [hunk 5]
```diff
@@ -19,14 +19,19 @@ function UAGBImageEdit( props ) {
 		attributes,
 		name,
 		attributes: { UAGHideDesktop, UAGHideTab, UAGHideMob },
-		deviceType
+		deviceType,
+		context
```
Comment (bsf-zanev) [inline src/blocks/image/edit.js:23]: Always use trailing comma.

```suggestion
		context,
```

File: src/blocks/info-box/edit.js [hunk 6]
```diff
@@ -34,7 +34,8 @@ const UAGBInfoBox = ( props ) => {
 		},
 		name,
 		clientId,
-		deviceType
+		deviceType,
+		context
```
Comment (bsf-zanev) [inline src/blocks/info-box/edit.js:38]: Always use trailing comma,

```suggestion
		context,
```

**Review by bsf-zanev — COMMENTED** [review]
Added a few suggestions.

**Review by aapatre — APPROVED** [review]
Pre-approved, please resolve the above

File: readme.txt [hunk 7]
```diff
@@ -166,6 +166,12 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= 2.6.8 - TUESDAY, 20th JUNE 2023 =
+* Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
+* Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.
+* Fix: Form - Resolved an issue where the submit button is visible on the frontend when no preset from the variation picker is selected.
```
[the remark this answers is not in the document]
Comment (bsf-zanev) [reply readme.txt:172]: I've updated this changelog entry [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.

File: src/blocks/container/edit.js (hunk 2 above)
Comment (bsf-zanev) [reply src/blocks/container/edit.js:44]: This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.

File: src/blocks/advanced-heading/edit.js (hunk 3 above)
Comment (bsf-zanev) [reply src/blocks/advanced-heading/edit.js:27]: This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.

File: src/blocks/buttons-child/edit.js (hunk 4 above)
Comment (bsf-zanev) [reply src/blocks/buttons-child/edit.js:26]: This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.

File: src/blocks/image/edit.js (hunk 5 above)
Comment (bsf-zanev) [reply src/blocks/image/edit.js:23]: This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.

File: src/blocks/buttons-child/styling.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/buttons-child/styling.js:342]: @bsf-zanev 2nd approach looks good to me. Could you please implement same in [this](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283) PR?

Comment (bsf-zanev) [reply src/blocks/buttons-child/styling.js:342]: The cleaner condition as stated in the apporach above has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.

File: languages/ultimate-addons-for-gutenberg.pot [hunk 8]
```diff
@@ -2,16 +2,16 @@
...
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"POT-Creation-Date: 2023-06-14T07:00:04+00:00\n"
+"POT-Creation-Date: 2023-06-19T13:25:52+00:00\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"X-Generator: WP-CLI 2.8.1\n"
+"X-Generator: WP-CLI 2.6.0\n"
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply languages/ultimate-addons-for-gutenberg.pot:14]: Please review - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3284

[the remark this answers is not in the document]
Comment (bsf-zanev) [reply languages/ultimate-addons-for-gutenberg.pot:14]: Approved. This comment will be resolved once that PR is merged.

CC - @adi3890.

**Review by AbhijitNage123 — APPROVED** [review]

File: languages/ultimate-addons-for-gutenberg.pot (hunk 8 above)
[the remark this answers is not in the document]
Comment (adi3890) [reply languages/ultimate-addons-for-gutenberg.pot:14]: Not significant

**Review by bsf-zanev — APPROVED** [review]

**Pull request merged**